src/main/java/ch/uzh/ifi/hase/soprafs22/rest/dto/CardPostDTO.java

```diff
@@ -1,6 +1,7 @@
 package ch.uzh.ifi.hase.soprafs22.rest.dto;
 
-public class BlackCardPostDTO {
```

Comment:
Not sure if we want to make a generaly CardPostDTO, if we want to later implement Black Cards with 2 "blanks" then we would need BlackCard and WhiteCardDTO's

Reply:
I don't see a problem there, but that's something that is very easily done, so I leave it for now.